terminal.go

```diff
@@ -74,6 +74,7 @@ const (
 
 	defaultServerTimeout  = 10 * time.Second
 	defaultConnectTimeout = 15 * time.Second
+	defaultRPCTimeout     = 1 * time.Minute
```

Comment:
The actual default timeout used in `lndclient` `lnd_services.go` is 30 seconds:
https://github.com/lightninglabs/lndclient/blob/d89c44c7ef25fdf72c901876b6961ebee7d0c133/lnd_services.go#L30

Would we like to mimic that in `litd`, or would we specifically like to set it to 1 minute in `litd`?

Reply:
Yes, this is an intentional bump. 30 seconds probably is too low already, especially looking at your other comment. Perhaps we should bump it even more?

Reply:
Thanks for clarifying! 

> Perhaps we should bump it even more?

Yeah maybe it'd make sense to bump it to 2 or 3 minutes even. Though I'd be ok with setting this default to 1 minute for now, and adjust it in future if needed. Feel free to set this to whatever you'd prefer of those alternatives :)!